edge-hub/src/Microsoft.Azure.Devices.Edge.Hub.Core/ConnectionManager.cs

```diff
@@ -97,6 +97,14 @@ public async Task<Option<ICloudProxy>> GetCloudConnection(string id)
                 .Map(c => (ICloudProxy)new RetryingCloudProxy(id, () => this.TryGetCloudConnection(id), c));
         }
 
+        public async Task<Try<ICloudProxy>> GetCloudConnectionTry(string id)
```

Comment:
```suggestion
        public async Task<Try<ICloudProxy>> TryGetCloudConnection(string id)
```

Reply:
renamed it to TryGetCloudConnection to be consistent with other GetCloudConnection.